src/test/resources/fixtures/entries.ttl

```diff
@@ -6,12 +6,16 @@
       register-metadata:entry-timestamp-field
               "2016-03-02T02:03:04Z" ;
       register-metadata:item-resource
-              <http://register.test.register.gov.uk/item/sha-256:63e5a0453b088e39265ca9f20fd03e2b206422e32989649adaca84426b531cd7> .
+              <http://register.test.register.gov.uk/item/sha-256:63e5a0453b088e39265ca9f20fd03e2b206422e32989649adaca84426b531cd7> ;
+      register-metadata:key-field
+              "value2" .
 
 <http://register.test.register.gov.uk/entry/1>
       register-metadata:entry-number-field
               "1" ;
       register-metadata:entry-timestamp-field
               "2016-03-01T01:02:03Z" ;
       register-metadata:item-resource
-              <http://register.test.register.gov.uk/item/sha-256:877d8bd1ab71dc6e48f64b4ca83c6d7bf645a1eb56b34d50fa8a833e1101eb18> .
+              <http://register.test.register.gov.uk/item/sha-256:877d8bd1ab71dc6e48f64b4ca83c6d7bf645a1eb56b34d50fa8a833e1101eb18> ;
+      register-metadata:key-field
```

Comment:
this doesn't correspond to a valid target in the specification:
http://openregister.github.io/specification/#key-field